http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -82,104 +204,226 @@ For the particular case of the HTTP connector, this means that it's only a good
+----
+<http:request method="GET" config-ref="HTTP_Request_configuration" path="/customer/{customerId}">
+			<http:uri-params ><![CDATA[#[[{'customerID' : '20'}]]]]></http:uri-params>
+		</http:request>
+----
 
-=== Dynamic Parameters and Headers
+When the Mule app runs, it performs a `GET` request to: +
 
-If, at design time, you don't know how many query parameters or headers a request might need, use DataWeave and a variable map to dynamically assign parameters or headers to the request.
+`+http://www.example.com/customer/20+`.
 
-For example, you create a map variable named `customMap` and assign the variable a map of values using DataWeave, then use that variable to set up the headers of your request:
+== Configure Dynamic Parameters and Headers
 
-`#[vars.customMap ++ {'HeaderName1' : 'HeaderValue1'}]`
+If during design time, you don't know how many parameters or headers a request might need, use DataWeave expressions and a variable map to dynamically assign parameters or headers to the HTTP request.
+
+To set headers dynamically, create a variable map and assign the variable values using DataWeave. Then, use that variable to set up the headers of your request. For example, with a variable map named `customMap`, you set up the following headers for the request: `#[vars.customMap ++ {'HeaderName1' : 'HeaderValue1'}]`
 
 To set URI parameters dynamically, use a DataWeave expression that returns a map of the parameters. For example:
 
+//Need to clarify this section with developers, as it is not a full DW expression
 * Set Path to: `/test/{p1}/{p2}`
 * URI Parameter names: p1 and p2
 * URI Parameter value: `#[vars.customMap]`
-* Before the request, assuming p1 already set: `#[vars.customMap ++ {'p2': 'customer'}]`
 
-The connector resolves parameters for each request, and evaluates DataWeave expressions in the context of the current message, in the order specified in the request. If the same parameter is defined more than once, the latest value is used.
+Before the request, assuming p1 already set: `#[vars.customMap ++ {'p2': 'customer'}]`
+
+HTTP Connector resolves parameters for each request, and evaluates DataWeave expressions in the context of the current message, in the order specified in the request. If the same parameter is defined more than once, the latest value is used.
 
+== Send Form Parameters
 
-=== Sending Form Parameters in a POST Request
+To send form parameters in a POST request, follow these steps:
 
-To send parameters in a POST request, In *General > Request*, select the *POST* method.
-In *Body*, construct the payload of the Mule message as `application/x-www-form-urlencoded` with the names and the values of the parameters to send. For example:
+. In Studio, select the HTTP *Request* operation from your flow.
+. In *General*, set *Method* to *POST*.
+. In *Body*, construct the payload of the Mule message as `application/x-www-form-urlencoded` with the names and the values of the parameters to send, for example:
 
```

Comment:
In this topic set I have seen `POST`, POST, and *POST*. Please ensure that these representations are appropriate to their contexts: syntax, name, and UI label, respectively.